notebooks/00_quick_start/xdeepfm_single_node.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
I would recommend to directly call the notebooks xdeepfm.ipynb and dkn.ipynb. The single node thing was because we have an algo called SAR, and in one of its implementations we call it single node. But it was specific for that algo

Reply:
OK, I will do that